src/statuses/server.clj

```diff
@@ -19,18 +20,29 @@
       wrap-not-modified
       wrap-stacktrace))
 
-(defn -main [& m]
-  (cfg/init! (or (first m) "config.clj"))
-  (println "Configuration: " (config))
-  (persistence/init! (config :database-path) (config :save-interval))
-  (println "Starting server on host"  (config :host)
-           "port" (config :http-port)
-           "in mode" (config :run-mode))
-  (run-jetty
-    (if (= (config :run-mode) :dev)
-      (wrap-reload app)
-      app)
-    {:host (config :host)
-     :port (config :http-port)
-     :join? false}))
+
+(def cli-options
+  [["-p" "--port PORT" "Port number" :parse-fn #(Integer/parseInt %)]
+  ["-h" "--host HOST" "Hostname"]
+  ["-c" "--conf ConfigurationFile" "The Location of the Configuration File"]])
+
+(defn -main [& args]
+  (let [{:keys [options arguments errors summary]} (parse-opts args cli-options)]
+    (cfg/init! (or (:conf options) "config.clj"))
```

Comment:
This is not backwards compatible. Before your change the configuration file could be provided as argument to the jar. This should not be broken as some users depend on this.